### PR TITLE
Update Microsoft.VMwareCloudSimple provider schema to Cloudsimple 44 release

### PR DESCRIPTION
**Pull request by mpoliako**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json [hunk 1]
```diff
@@ -433,6 +433,119 @@
...
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.VMwareCloudSimple/locations/{regionId}/privateClouds/{pcName}/customizationPolicies/{customizationPolicyName}": {
+      "get": {
+        "description": "Returns resource pool templates by its name",
```
Comment (shivam-msft) [inline specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json]: Please check if the description is accurate.

File: specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json [hunk 2]
```diff
@@ -433,6 +433,119 @@
...
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "enum": [
+              "linux",
+              "windows",
+              "other"
+            ],
+            "type": "string",
+            "description": "If specified response will be filtered by guest os type",
```
Comment (KrisBash) [inline specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json]: Per Microsoft REST API guidelines, this should follow odata $filter syntax: https://github.com/Microsoft/api-guidelines/blob/master/Guidelines.md#97-filtering

Here's a relevant example: https://docs.microsoft.com/en-us/rest/api/batchservice/odata-filters-in-batch#list-support-images

File: specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json [hunk 3]
```diff
@@ -433,6 +433,119 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/CustomizationPolicy"
+            }
+          },
+          "default": {
+            "description": "BadRequest",
```
Comment (KrisBash) [inline specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json]: Nit - Description shouldn't be specific to 400.

File: specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json [hunk 4]
```diff
@@ -2393,14 +2707,91 @@
         },
         "nodes": {
           "description": "total nodes purchased",
-          "type": "integer"
+          "type": "integer",
+          "readOnly": true
```
Comment (KrisBash) [inline specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json]: I assume this is a client side bugfix and not a (breaking) change of a property to read-only?

**Review by KrisBash — CHANGES_REQUESTED** [review]
Please see comment on $filter

File: specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json (hunk 4 above)
Comment (mpoliako) [reply specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json]: this property was never modifiable from client side. So this is not breaking change

**Review by KrisBash — APPROVED** [review]
LGTM thanks

**Comment by mpoliako** [comment]
@erich-wang @KrisBash 
hi guys, could you please merge this pull request,
we have rollout this version of API to production
Thank you

**Comment by mpoliako** [comment]
@erich-wang @KrisBash @jhendrixMSFT 
Hi could you please merge this request, we already have this version of API in prod

Thank you

**Review by jhendrixMSFT — APPROVED** [review]

**Comment by mpoliako** [comment]
@jhendrixMSFT Thank you

**Pull request merged**